src/main/java/com/shrralis/ssdemo1/controller/system/GeneralRestController.java

```diff
@@ -0,0 +1,18 @@
+package com.shrralis.ssdemo1.controller.system;
+
+import com.shrralis.tools.model.JsonError;
+import com.shrralis.tools.model.JsonResponse;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * @author shrralis (https://t.me/Shrralis)
+ * @version 1.0 Created 12/23/17 at 9:06 PM
+ */
+@RestController
+public class GeneralRestController {
+	@RequestMapping("accessDenied")
+	public JsonResponse accessDenied() {
+		return new JsonResponse(JsonError.Error.ACCESS_DENIED);
```

Comment:
Do you have separate endpoint for accessDenied? I would expect that it will be some exception handler

Reply:
O.K.